upcase_booklist_app/data/author.py

```diff
@@ -0,0 +1,72 @@
+
+authors = [
+    {
+        "author_id": 1,
+        "name": "Eric Matthes",
+        "bio": "Eric Matthes is a high school math and science teacher living in Alaska who teaches an Introduction to Programming class in Python.",
+        "books": [
```

Comment:
You'll notice that this book data is redundant with your `books` data structure, so we'll want to "normalize" it in the future, and databases are perfect for this.